### DISCUSSION TITLE
Get window position

### DISCUSSION THREAD

**Discussion by connor-devuk** [NativePHP/laravel#311] [Q&A] [answered]
Hey guys,

I was wondering if there is a way to get a position and height/width of a window, the main purpose for this is when opening a new window right now it opens on my 1st screen whilst the app is on the third, and I want it to open right next to the main window any help or little hacks todo this would be greatly appreciated thank you :)

**Comment by simonhamp** [comment]
[Yes, you can](https://nativephp.com/docs/1/the-basics/windows#retrieving-the-current-window)

**Comment by Tray2** [comment, marked as answer]
To get the position you can just do

```
$window = Window::current();
$window->x;
$windows->y;
```
